logging/fluentd/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM fluent/fluentd:v1.2.1-onbuild
+MAINTAINER Petr Kotas <petr.kotas@gmail.com>
+
+# Needed for the forwaring to work properly
+EXPOSE 24224/udp
```

Comment:
Is `udp` only reason for our own Docker image? If so, we can probably get around this by setting `heartbeat_type` config of the forward plugin to tcp, see https://docs.fluentd.org/v1.0/articles/out_forward#heartbeat_type

Reply:
Yes, the forward was sending broken messages without the udp exposed. Thanks for pointing the heartbeat will check on that.